app/views/hub/clients/show.html.erb

```diff
@@ -178,6 +178,12 @@
                       <span class="label-value"><%= "$#{benefits.advance_ctc_amount_received}" if benefits.advance_ctc_amount_received %></span>
                     </div>
                   <% end %>
+                  <% unless benefits.eitc_amount.nil? %>
+                    <div class="field-display">
+                      <span class="form-question">EITC estimated amount:</span>
+                      <span class="label-value"><%= "$#{benefits.eitc_amount}" %></span>
```

Comment:
Non-blocking: This could use the I18n number formatting.

Reply:
@alaroia-cfa like putting "EITC estimated amount:" in the yml files? I didn't put it in originally b/c there was some other text on the client profile that was left out.